app/components/editor-document-title.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if @readOnly}}
   <h1 class="au-c-app-chrome__title">
-    {{this.title}}
+    {{limit-content this.title 70}} 
```

Comment:
probably want some way to see the full title when read-only. (when not read-only, you can see it by editing it)